lidarrmetadata/chart.py

```diff
@@ -0,0 +1,171 @@
+"""
+Code for getting and parsing music charts (Billboard, itunes, etc)
+"""
+
+import billboard
+import pylast
+import requests
+
+from lidarrmetadata import config
+from lidarrmetadata import provider
+
+
+def get_apple_music_chart(count=10):
+    """
+    Gets and parses itunes chart
+    :param count: Number of results to return. Defaults to 10
+    :return: Chart response for itunes
+    """
+    URL = 'https://rss.itunes.apple.com/api/v1/us/apple-music/top-albums/all/{count}/explicit.json'.format(count=4 * count)
+    response = requests.get(URL)
+    results = response.json()['feed']['results']
+
+    search_provider = provider.get_providers_implementing(provider.AlbumNameSearchMixin)[0]
+
+    search_results = []
+    for result in results:
+        search_result = search_provider.search_album_name(result['name'], result['artistName'])
```

Comment:
Should we add a parameter to the search module to allow a LIMIT to be set in sql, would this save sql execution time?

Reply:
Good idea. I'll add that.